setup.cfg

```diff
@@ -122,10 +122,6 @@ disallow_incomplete_defs = False
 disallow_untyped_defs = False
 disallow_incomplete_defs = False
 
-[mypy-reconcile.dashdotdb_base]
-disallow_untyped_defs = False
-disallow_incomplete_defs = False
-
 [mypy-reconcile.dashdotdb_cso]
 disallow_untyped_defs = False
 disallow_incomplete_defs = False
```

Comment:
If you add type hints to `cso` you can remove this obsolete setting :)

Reply:
Removed :)